lib/cmsis_rtos_v1/cmsis_mempool.c

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (c) 2018 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <kernel_structs.h>
+#include <cmsis_os.h>
+
+/**
+ * @brief Create and Initialize a memory pool.
+ */
+osPoolId osPoolCreate(const osPoolDef_t *pool_def)
+{
+	if (_is_in_isr()) {
+		return NULL;
+	}
+
+	return (osPoolId)pool_def;
+}
+
+/**
+ * @brief Allocate a memory block from a memory pool.
+ */
+void *osPoolAlloc(osPoolId pool_id)
+{
+	osPoolDef_t *osPool = (osPoolDef_t *)pool_id;
+	void *ptr;
+
+	if (k_mem_slab_alloc((struct k_mem_slab *)(osPool->pool), &ptr, 100)
```

Comment:
Where is the 100 coming from? magic number?

Reply:
i'll make it a macro. It's the timeout parameter and putting it in Kconfig will be an overkill.